Apps/RaptureAPIServer/src/main/resources/rapture/config/rapture.cfg

```diff
@@ -2,3 +2,4 @@ ServerType: "webserver"
 Categories: "alpha"
```

Comment:
(Curtis could, because that's really "the demo server" perhaps so it makes sense to have app configuration there, but not for the boilerplate apiserver??)